test/plugin.test.js

```diff
@@ -323,17 +323,48 @@ QUnit.test('unrecognized error codes do not cause exceptions', function(assert)
 });
 
 QUnit.test('custom error details should override defaults', function(assert) {
```

Comment:
If someone provides a custom error message, we should not modify it (or append extra text to it). It might not make sense following their message and would probably confuse them.